lib/ship.rb

```diff
@@ -0,0 +1,22 @@
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
+    @health = 3
+    @sunk = false
+  end
+
+  def sunk?
```

Comment:
This method can be shortened to a single line -
`@health <= 0`

Reply:
I have updated the code!